'QMessageBox’ was not declared in this scope

I got this error trying to compile the current master (dbac1b9):

```
x86_64-pc-linux-gnu-g++ -c -O2 -pipe -march=haswell -fvisibility=hidden -Wall -Wextra -fstack-protector -fPIE -U_FORTIFY_SOURCE -g -I../mumble_proto -isystem ../mumble_proto -I../../3rdparty/celt-0.7.0-src/libcelt -isystem ../../3rdparty/celt-0.7.0-src/libcelt -std=gnu++1z -pthread -Wall -Wextra -D_REENTRANT -fPIC -DPLUGIN_PATH=/usr/lib64/mumble -DRESTRICT=__restrict__ -DSNAPSHOT_BUILD -DMUMBLE_VERSION_STRING=1.4.0 -DMUMBLE -DUSE_OVERLAY -DUSE_MANUAL_PLUGIN -DNO_CRASH_REPORT -DUSE_OPUS -DUSE_RNNOISE -DUSE_NO_TTS -DUSE_PULSEAUDIO -DUSE_DBUS -DNO_UPDATE_CHECK -D_REENTRANT -DQT_SVG_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_SQL_LIB -DQT_XML_LIB -DQT_CONCURRENT_LIB -DQT_DBUS_LIB -DQT_CORE_LIB -I. -I../../src -I. -I../mumble_proto -I../../3rdparty/arc4random-src -I../../3rdparty/qqbonjour-src -I../../3rdparty/smallft-src -Iwidgets -I../../3rdparty/celt-0.7.0-src/libcelt -I../../3rdparty/rnnoise-src/include -isystem /usr/include/opus -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtSvg -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtNetwork -isystem /usr/include/qt5/QtSql -isystem /usr/include/qt5/QtXml -isystem /usr/include/qt5/QtConcurrent -isystem /usr/include/qt5/QtDBus -isystem /usr/include/qt5/QtCore -I../../debug/.moc/mumble -isystem /usr/include/libdrm -I../../debug/.ui/mumble -I/usr/lib64/qt5/mkspecs/linux-g++ -o ../../debug/.obj/mumble/ListenerLocalVolumeDialog.o ListenerLocalVolumeDialog.cpp
NetworkConfig.cpp: In member function ‘void NetworkConfig::on_qcbAutoUpdate_stateChanged(int)’:
NetworkConfig.cpp:158:3: error: ‘QMessageBox’ was not declared in this scope; did you mean ‘QMessageLogger’?
  158 |   QMessageBox msgBox;
      |   ^~~~~~~~~~~
      |   QMessageLogger
NetworkConfig.cpp:159:3: error: ‘msgBox’ was not declared in this scope
  159 |   msgBox.setText(QObject::tr("<p>You're using a Mumble version that <b>explicitly disabled</b> update-checks.<p>"
      |   ^~~~~~
NetworkConfig.cpp:166:18: error: ‘QMessageBox’ is not a class, namespace, or enumeration
  166 |   msgBox.setIcon(QMessageBox::Warning);
      |                  ^~~~~~~~~~~
make[2]: *** [Makefile.Debug:3559: ../../debug/.obj/mumble/NetworkConfig.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/var/tmp/portage/media-sound/mumble-9999/work/mumble-9999/src/mumble'
make[1]: *** [Makefile:42: debug] Error 2
make[1]: Leaving directory '/var/tmp/portage/media-sound/mumble-9999/work/mumble-9999/src/mumble'
make: *** [Makefile:181: sub-src-mumble-make_first-ordered] Error 2

```

--- a/src/mumble/NetworkConfig.cpp
+++ b/src/mumble/NetworkConfig.cpp
@@ -13,6 +13,10 @@
 #include <QtNetwork/QNetworkAccessManager>
 #include <QSignalBlocker>
 
+#ifdef NO_UPDATE_CHECK
+	#include <QMessageBox>
+#endif
+
 // We define a global macro called 'g'. This can lead to issues when included code uses 'g' as a type or parameter name (like protobuf 3.7 does). As such, for now, we have to make this our last include.
 #include "Global.h"
 

